### PR TITLE
[CLN] payment: update docstrings for Sphinx's autodoc

### PR DESCRIPTION
**Pull request by AntoineVDV**
task-2804999

See also:
- https://github.com/odoo/documentation/pull/2782

### CODE REVIEW

**Comment by AntoineVDV** [comment]
@robodoo override=ci/security
Reason: False positive triggered by a typo fix in the comment.
Link to ci/security build (16.0): https://runbot.odoo.com/runbot/build/19807409

File: addons/payment/models/payment_transaction.py [hunk 1]
```diff
@@ -866,21 +871,29 @@ def _cron_finalize_post_processing(self):
...
+
+        :return: None
+        """
+        self._reconcile_after_done()
+        self._log_received_message()  # 2nd call to link the created account.payment in the chatter
+        self.is_post_processed = True
+
     def _reconcile_after_done(self):
-        """ Post relevant fiscal documents and create missing payments.
+        """ Perform compute-intensive operations on related documents.
```
Comment (Feyensv) [inline addons/payment/models/payment_transaction.py:893]: compute-intensive ? :D

File: addons/payment/models/payment_transaction.py [hunk 2]
```diff
@@ -865,21 +871,28 @@ def _cron_finalize_post_processing(self):
...
                 _logger.exception(
                     "encountered an error while post-processing transaction with reference %s:\n%s",
                     tx.reference, e
                 )
                 self.env.cr.rollback()
 
+    def _finalize_post_processing(self):
+        """ Trigger the final post-processing tasks and mark the transactions as post-processed.
+
+        :return: None
```
Comment (Demesmaeker) [inline addons/payment/models/payment_transaction.py:886]: return ?

**Review by Demesmaeker — APPROVED** [review]
:duck:

**Comment by AntoineVDV** [comment]
@robodoo r+

**Pull request closed without merging**